src/components/GenericPagesComponents/PostImageWithAltText/PostImageWithAltText.tsx

```diff
@@ -11,7 +11,7 @@ const PostImageWithAltText = (
     return null;
   }
 
-  return <CMSImage image={props.value} $mt={80} $mb={64} />;
+  return <CMSImage image={props.value} $mt={24} $mb={24} />;
```

Comment:
This is change how blogs will look, is it a change we want ?

Reply:
I'll double check with Monika, but my feeling is that all the blogs should have the same styling for consistency